src/chapters/06/main.tex

```diff
@@ -25,44 +25,66 @@ \section{Problem}\label{sec:problem}
         profit.
 \end{itemize}
 
-The expected profits are given in Table~\ref{tbl:profit-of-taxi-companies}.
+The expected profits for Taxi Company A are given in Table~\ref{tbl:profit-of-taxi-companies}.
+The profits for Taxi Company B are the same.
 
+\newcolumntype{P}[1]{>{\centering\arraybackslash}p{#1}}
 \begin{table}[!hbtp]
     \begin{center}
-    \begin{tabular}{l|ccc}
-        \diagbox{Taxi numbers}{Other company taxi numbers} & 1     & 2      & 3\\
+    \begin{tabular}{cl|P{0.08\textwidth}P{0.08\textwidth}P{0.08\textwidth}}
+                                                                  &    & \multicolumn{3}{c}{Taxi Company B}\\
+                                                                  &    & 1               & 2                 & 3\\
         \toprule
-        1                                                  & 1     & \(\frac{1}{2}\) & \(\frac{1}{3}\)\\
-        2                                                  & \(\frac{3}{2}\)   & \(\frac{19}{20}\)& \(\frac{1}{2}\)\\
-        3                                                  & \(\frac{5}{3}\) & \(\frac{4}{5}\)  & \(\frac{17}{20}\)\\
+        \multirow{3}{*}{\rotatebox[origin=c]{90}{Taxi Company A}} & 1  & 1               & \(\frac{1}{2}\)   & \(\frac{1}{3}\)\\[4.5mm]
+                                                                  & 2  & \(\frac{3}{2}\) & \(\frac{19}{20}\) & \(\frac{1}{2}\)\\[4.5mm]
+                                                                  & 3  & \(\frac{5}{3}\) & \(\frac{4}{5}\)   & \(\frac{17}{20}\)\\[4.5mm]
     \end{tabular}
     \end{center}
-    \caption{Profits (in GBP per hour) of a given company based on
-    their vehicle numbers and the other companies vehicle numbers.}
+    \caption{Profits (in GBP per hour) of a Taxi Company A based on
+    their vehicle numbers and the vehicle numbers of Taxi Company B.}
     \label{tbl:profit-of-taxi-companies}
 \end{table}
 
+% TODO I tried to make the table look better and more readable. Feel free to revert. I don't like that the horizontal and vertical lines don't touch. The column names were confusing, so I made them refer to specific companies now.
```

Comment:
I don't understand what was confusing about the column names (we can talk about that on Friday) but not terribly fussed about the table itself so if you like it better then I'm sure it's fine.